library/lint.xml

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why you provided these empty Lint files? I think that if you don't want to change any lint rules, you don't need any file.

Reply:
So if they will be needed in future, they will be there